nansat/exporter.py

```diff
@@ -300,8 +311,27 @@ def _create_dimensions(self, nc_inp, nc_out, time):
 
         return grid_mapping_name, grid_mapping_var_name
 
-    def _post_proc_thredds(self, tmp_filename, out_filename, bands, band_metadata, time, global_metadata):
-        """Post processing of file for THREDDS (add time variable and metadata)"""
+    def _post_proc_thredds(self,
```

Comment:
General comments for the function: 
* The function returns 0 which seems to be not needed
* Please refactor some variable names within the function according to the pep8

Reply:
Done